README.md

```diff
@@ -125,7 +125,7 @@ gcloud container clusters delete onlineboutique \
 
 ## Automated Deployment
 
-Automated deployment for the Online Boutique sample application is now supported in this repository. Instructions on using Terraform to replicate the **Quickstart (GKE)** steps can be found [here](https://github.com/GoogleCloudPlatform/microservices-demo/tree/main/terraform).
+Automated deployment for the Online Boutique sample application is now supported in this repository. Instructions for using Terraform to replicate the **Quickstart (GKE)** steps can be found inside [/main/terraform/README.md](https://github.com/GoogleCloudPlatform/microservices-demo/tree/main/terraform).
```

Comment:
My rationale for change: Avoid "here" for links because it doesn't provide enough meaning about the link. [Source](https://youtu.be/sXj0r4z5B5M?t=773).